### ISSUE TITLE
Null pointer exception if you pass wrong arguments to /mvgamerule

### ISSUE THREAD

**Issue by jackorp** [Multiverse/Multiverse-Core#2107]
Hi, I executed a `/mvgamerule arena_2 doMobSpawning false` command but got it wrong. I passed in world name which I presume resulted in pointer null exception. It would be nice if it produced normal "this is how you use the command" message. `arena_2` is my valid world.

env:
OpenJDK 8
Multiverse-core 4.1.0
running on Paper-196
~~~
[22:23:31] [Server thread/INFO]: Mistr25 issued server command: /mvgamerule arena_2 doMobSpawning false
[22:23:31] [Server thread/WARN]: java.lang.NullPointerException
[22:23:31] [Server thread/WARN]: 	at com.onarandombox.MultiverseCore.commands.GameruleCommand.runCommand(GameruleCommand.java:67)
[22:23:31] [Server thread/WARN]: 	at com.onarandombox.commandhandler.CommandHandler.checkAndRunCommand(CommandHandler.java:296)
[22:23:31] [Server thread/WARN]: 	at com.onarandombox.commandhandler.CommandHandler.processFoundCommands(CommandHandler.java:143)
[22:23:31] [Server thread/WARN]: 	at com.onarandombox.commandhandler.CommandHandler.locateAndRunCommand(CommandHandler.java:93)
[22:23:31] [Server thread/WARN]: 	at com.onarandombox.MultiverseCore.MultiverseCore.onCommand(MultiverseCore.java:915)
[22:23:31] [Server thread/WARN]: 	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[22:23:31] [Server thread/WARN]: 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
[22:23:31] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:739)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PlayerConnection.handleCommand(PlayerConnection.java:1800)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1608)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PacketPlayInChat.a(PacketPlayInChat.java:47)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PacketPlayInChat.a(PacketPlayInChat.java:5)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.aX(MinecraftServer.java:1013)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:1006)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:990)
[22:23:31] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:923)
[22:23:31] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
~~~

**Comment by benwoo1110** [comment]
I think the wiki explains help to use the commands already. If command is excited wrongly it’s natural to have an error. However, I do agree there should be better error handling.

**Comment by jackorp** [comment]
I agree. At least saying something like "you got it wrong" is better than null pointer exception :)

**Comment by nicegamer7** [comment]
This was fixed in #2227.
